src/screens/Ticketing/Purchase/Overview/index.tsx

```diff
@@ -242,6 +239,49 @@ export const createTravelDateText = (
       )
     : t(PurchaseOverviewTexts.travelDate.now);
 };
+
+/**
+ * Get the default user profiles with count. If a default user profile has been
+ * selected in the preferences that profile will have a count of one. If no
+ * default user profile preference exists then the first user profile will have
+ * a count of one.
+ */
+const useDefaultUserProfilesWithCount = (
+  userProfiles: UserProfile[],
+  preassignedFareProduct: PreassignedFareProduct,
+) => {
+  const {
+    preferences: {defaultUserTypeString},
+  } = usePreferences();
+
+  const isDefaultProfile = useCallback(
+    (u: UserProfile, index: number, filteredProfiles: UserProfile[]) => {
+      if (
+        defaultUserTypeString &&
+        filteredProfiles.some((u) => u.userTypeString === defaultUserTypeString)
```

Comment:
Shadowed variable. Technically not wrong and refers to the same type here, but could be confusing?